docs/guidance/map/framework_agreements.md

```diff
@@ -106,46 +107,43 @@ The following guidance describes how to model the different stages of a framewor
 
 ### Award of a procurement contract without second-stage competition
 
-* Create a release with a **new** `ocid` and add 'award' and 'contract' to the `tag`
-* [Relate the second stage to the first stage](#relate-the-second-stage-to-the-first-stage)
-* [Add a buyer](#add-a-buyer)
-* Set `tender.id` to the `award.id`, set `tender.competitive` to `false`, and set `tender.procuringEntity` if appropriate
-* [Add an award, contract and supplier](#add-an-award-contract-and-supplier)
+* Create a release with a **new** `ocid` and add 'award' to the `tag`.
+* [Relate the second stage to the first stage](#relate-the-second-stage-to-the-first-stage).
+* [Add a buyer](#add-a-buyer).
+* Set `tender.id` to the `award.id`, set `tender.competitive` to `false`, and set `tender.procuringEntity` if appropriate.
+* [Add an award and supplier](#add-an-award-and-supplier).
 
 ### Invitation to participate in a second-stage competition
 
-* Create a release with a **new** `ocid` and add 'tender' to the `.tag` array
-* [Relate the second stage to the first stage](#relate-the-second-stage-to-the-first-stage)
-* [Add a buyer](#add-a-buyer)
-* Populate the `tender` section, setting `tender.procurementMethod` to the same value as in the first stage, and `tender.competitive` to `true`
+* Create a release with a **new** `ocid` and add 'tender' to the `.tag` array. 
+* [Relate the second stage to the first stage](#relate-the-second-stage-to-the-first-stage).
+* [Add a buyer](#add-a-buyer).
+* Populate the `tender` section, setting `tender.procurementMethod` to the same value as in the first stage, and `tender.competitive` to `true`.
 
 ### Award of a procurement contract resulting from a second-stage competition
 
-* Create a release with the **same** `ocid` as the tender release *for the invitation to participate in a second-stage competition*, and add 'award' and 'contract' to the `tag`
-* [Add an award, contract and supplier](#add-an-award-contract-and-supplier)
-
-### Updating the contract value, period or items
-
-* If the initial contract value, period or items are subsequently updated, populate the contract's `.value`, `.period` and `.items` with the updated values.
+* Create a release with the **same** `ocid` as the tender release *for the invitation to participate in a second-stage competition*, and add 'award' to the `tag`.
+* [Add an award and supplier](#add-an-award-and-supplier).
 
 ### Common operations
 
 #### Relate the second stage to the first stage
 
 * Add a `RelatedProcess` object to the `relatedProcesses` array, set its `.id` (to '1', for example), add 'framework' to its `.relationship` array, set its `.scheme` to 'ocid' and set its `.identifier` to the `ocid` of the invitation to participate in the framework agreement.
+* Note: second-stage contracting processes ought to be always related to the first-stage contracting process, never to other second-stage contracting processes.
```

Comment:
I think this second note is not needed, as the preceding sentence is explicitly about relating to "the invitation to participate in the framework agreement".

```suggestion
```

Reply:
Sounds good, done.